src/popup/components/TransactionOverview.vue

```diff
@@ -59,26 +61,28 @@ export default defineComponent({
 
     const { getSdk } = useSdk({ store: root.$store });
 
+    const { tx } = props;
```

Comment:
Destructuring props is a bad thing because the reactivity is lost. I see that in this case this is not a problem, but still... I think we should not do it for the sake of having some common rules.

https://stackoverflow.com/questions/64926354/vue-3-watch-doesn-t-work-if-i-watch-a-destructured-prop

In the future we need to add ESLint rule for this.